src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService x = new UserService();
```

Comment:
```suggestion
        UserService user = new UserService();
```
It is better to move this line to the class level. Otherwise, a new object of the type UserService will be created each time this method is called